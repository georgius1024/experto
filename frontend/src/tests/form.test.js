import React from 'react'
import renderer from 'react-test-renderer'
import ChatForm from '../components/chat/form'
describe('ChatForm component', () => {
  test('it matches the snapshot', () => {
    const component = renderer.create(<ChatForm />)
    expect(component.toJSON()).toMatchSnapshot()
  })
})
