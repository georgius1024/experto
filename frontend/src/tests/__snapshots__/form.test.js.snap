// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ChatForm component it matches the snapshot 1`] = `
<form
  className=""
  onSubmit={[Function]}
>
  <div
    className="form-group"
  >
    <input
      className="form-control"
      onChange={[Function]}
      placeholder="Чат"
      type="text"
      value=""
    />
  </div>
</form>
`;
